spec/consensus/proposer-based-timestamp/pbts-sysmodel_002_draft.md

```diff
@@ -0,0 +1,222 @@
+# Proposer-Based Time v2 - Part I
+
+## System Model
+
+#### **[PBTS-CLOCK-NEWTON.0]**
+
+There is a reference Newtonian real-time `t` (UTC).
+
+No process has direct access to this reference time, used only for specification purposes.
+
+### Synchronized clocks
+
+Processes are assumed to be equipped with synchronized clocks.
+
+This requires processes to periodically synchronize their local clocks with an
+external and trusted source of the time (e.g. NTP servers).
+Each synchronization cycle aligns the process local clock with the external
+source of time, making it a *fairly accurate* source of real time.
+The periodic (re)synchronization aims to correct the *drift* of local clocks,
+which tend to pace slightly faster or slower than the real time.
+
+To avoid an excessive level detail in the parameters and guarantees of
+synchronized clocks, we adopt a single system parameter `PRECISION` to 
+encapsulate the potential inaccuracy of the synchronization mechanisms,
+and drifts of local clocks from real time.
+
+#### **[PBTS-CLOCK-PRECISION.0]**
+
+There exists a system parameter `PRECISION`, such that
+for any two processes `p` and `q`, with local clocks `C_p` and `C_q`,
+that read their local clocks at the same real-time `t`,  we have:
+
+- If `p` and `q` are equipped with synchronized clocks, then `|C_p(t) - C_q(t)| < PRECISION`
+
+`PRECISION` thus bounds the difference on the times simultaneously read by processes
+from their local clocks, so that their clocks can be considered synchronized.
+
+#### Accuracy
+
+The [previous system model][v1] included a second clock-related parameter, `ACCURACY`,
+that relates the values read by processes from their synchronized clocks with the real time.
+
+This parameter was removed in this version for two reasons:
+
+- Clock accuracy is hard, if possible at all, to assess in distributed systems
+- The adoption of an `ACCURACY` parameter renders the `PRECISION` parameter redundant:
+if the accuracy of processes clocks is bound by `ACCURACY`,
+then the clocks precision must be bound by `2 * ACCURACY`.
+
+The adoption of an `ACCURACY` was intended to formalize the relation between a block time and the real time.
+We observe, however, that clients will compare the block time to their local time,
+or to the time they retrieve from a trusted source of time, not to a physical clock.
```

Comment:
> not to a physical clock

I'm not totally sure I understand what this phrase means since their local time is generated from a physical clock. I think this sentence is attempting to describe the fact that processes local clocks are not perfectly accurate, is that roughly correct?

Reply:
I don't like this description either. I will try to fix it.

Reply:
Please take a look on the new version.